ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__multi-nightly.yaml

```diff
@@ -983,6 +983,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-internal-registry
+- as: azure-ipi-fullyprivate-oidc-amd-f14
+  cron: 51 20 4,18 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      EXTRACT_MANIFEST_INCLUDED: "true"
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-cco-manual-workload-identity
+- as: azure-ipi-fullyprivate-oidc-arm-f14-destructive
```

Comment:
```suggestion
- as: azure-ipi-fullyprivate-oidc-arm-f28-destructive
```

Reply:
Thanks for reviewing so fast. I have a card in debug test, this PR still has an issue I haven't fixed, I'll update it together later.

Reply:
btw, when you are going to update it, can you also help add a z-stream upgrade job in openshift-openshift-tests-private-release-4.16__multi-nightly-4.16-upgrade-from-stable-4.16.yaml for this install profile.